src/constants/deviceParameters/rx2SpreadFactors.ts

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
do we really need this subdir?

if yes then deviceParameters.ts should be renamed to index.ts a moved here as well?

Reply:
moved to root constants directory